### PR TITLE
add manhuaus

### PR DESCRIPTION
**Pull request by MareksKing**
Request to add manhuaus

### CODE REVIEW

File: src/pages/ManhuaUS/main.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import { util } from "chai";
```
Comment (henrik9999) [inline src/pages/ManhuaUS/main.ts:1]: Why is this imported

File: src/pages/playerUrls.js [hunk 2]
```diff
@@ -882,4 +882,8 @@ module.exports = {
   soraplay: {
     match: ['*://*.soraplay.xyz/*'],
   },
+  //ManhuaUS
+  manhuaus:{
```
Comment (henrik9999) [inline src/pages/playerUrls.js:886]: This is a manga page it should not need player urls

**Review by henrik9999 — CHANGES_REQUESTED** [review]

File: src/pages/ManhuaUS/main.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+import { pageInterface } from "../pageInterface";
+import { url } from "inspector";
+
+export const manhuaus: pageInterface = {
```
Comment (henrik9999) [inline src/pages/ManhuaUS/main.ts:5]: The name and export should be written ManhuaUS

**Comment by henrik9999** [comment]
`npm run lint:fix`

File: src/pages/ManhuaUS/meta.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+    "search": "https://manhuaus.com/?s={searchterm}&post_type=wp-manga&op=&author=&artist=&release=&adult=",
+    "urls": {
+        "match": ["*//manhuaus.com/*"]
```
Comment (lolamtisch) [inline src/pages/ManhuaUS/meta.json:4]: You forgot the `:`
["*://manhuaus.com/*"]

File: src/pages/ManhuaUS/main.ts [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    uiSelector(selector) {
+      j.$('div.c-page__content div.c-blog__heading')
+        .first()
+        .before(
+          j.html(
+            `<div id="malthing"><div id= "MALSyncheading" class="c-blog__heading style-2 font-heading"><h2 class="h4"> <i class="icon ion-ios-star"></i> MAL-Sync</h2></div>${selector}</div>`,
+          ),
+        );
+    },
+    list: {
```
Comment (lolamtisch) [inline src/pages/ManhuaUS/main.ts:55]: The list handling does not work

File: src/pages/ManhuaUS/main.ts [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+      const temp = episodePart.match(/chapter-\d+/gim);
+      if (!temp || temp.length === 0) return NaN;
+
+      return Number(temp[0].replace(/\D+/g, ''));
+    },
+    nextEpUrl(url) {
+      return j
+        .$(
+          'div.entry-header.header > div > div.select-pagination > div.nav-links > div.nav-next > a.next_page',
```
Comment (lolamtisch) [inline src/pages/ManhuaUS/main.ts:33]: Would be nice to make this less specific. Because a small page change would break this selector

**Review by lolamtisch — CHANGES_REQUESTED** [review]

**Pull request closed without merging**